src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue

```diff
@@ -252,12 +251,10 @@ function getPickedTokens(tokens: PoolToken[]) {
           <BalAssetSet :logoURIs="orderedTokenURIs(gauge)" :width="100" />
         </div>
       </template>
-      <template
-        #poolCompositionCell="{ pool, address, addedTimestamp, network }"
-      >
+      <template #poolCompositionCell="{ pool, address, addedTimestamp }">
         <div v-if="!isLoading" class="flex items-center py-4 px-6">
-          <div v-if="poolMetadata(pool.id, network)" class="text-left">
-            {{ poolMetadata(pool.id, network)?.name }}
+          <div v-if="poolMetadata(pool.id)" class="text-left">
```

Comment:
@alter-eggo As you can see I have removed second parameter network from `poolMetadata` function. This is the only place where we were using it and I don't think we need it but I'm not very familiar with this Gauge component and I'm not sure how to test it. Could you please double check?

Reply:
yeah, after @timjrobinson refactoring there is no need in network param